continuous-deployment/readme.md

```diff
@@ -1,7 +1,14 @@
 # Continuous Deployment
 
-Please see [CI/CD](../continuous-integration/CICD.md) while this page is awaiting contributions.
+Continuous Deployment can be thought of as a deployment pipeline where a piece of software needs to be validated before getting released anytime into production. It is a software engineering process where each functionality is pushed into production through automated deployments.
+
+Continuous deployments can be achieved by creating different environments in [Azure DevOps](https://docs.microsoft.com/en-us/aspnet/core/azure/devops/cicd) or by using multiple [GitHub workflows](https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow). These environments can be used for different reasons like some environments are used for development and testing while others are used for production.
+
+More details on environments and the steps for setting up different environments using GitHub workflows can be found [here](../continuous-deployment/pipelines/recipes/github-workflows/workflow-per-environment.md).
+
+For additional information and details on CI/CD please refer to this [link](../continuous-integration/CICD.md).
 
 ## Sections within Continuous Deployment
 
 * [Secrets Management](secrets-management/readme.md)
+* [Pipelines](pipelines/readme.md)
```

Comment:
I might add a link to the explanation of difference between Continuous Delivery and Deployment. Also, make sure the content is consistent on which one its describing. Example article here... https://www.atlassian.com/continuous-delivery/principles/continuous-integration-vs-delivery-vs-deployment

Delivery = Ship on time bound intervals. Also commonly referred to as Release Trains in larger dev shops
Deployment = Ship on commit all the way to prod.

Both are automated but, delivery will require someone to click a button to ship. Deployment lands in prod with absolutely no human intervention.